src/settings/OrderNumber.js

```diff
@@ -22,44 +14,61 @@ class OrderNumber extends Component {
 
   constructor(props) {
     super(props);
+
     this.configManager = props.stripes.connect(ConfigManager);
   }
 
   getInitialValues = (settings) => {
-    const value = settings.length && settings[0].value === 'true';
+    const value = settings.length === 0 ? '' : settings[0].value;
+    const defaultConfig = {
+      orderNumber: false,
```

Comment:
this name is not clear at all, IMHO. It's a spill-over from previous PR https://github.com/folio-org/ui-orders/pull/134#discussion_r245263010